tests/test_report.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env pytest -vs
+"""Tests for report."""
+
+# Standard Python Libraries
+import logging
+import os
+import sys
+from unittest.mock import patch
+
+# Third-Party Libraries
+import pytest
+
+# cisagov Libraries
+from pca_report_library import report
+
+div_params = [
+    (1, 1, 1),
+    (2, 2, 1),
+    (0, 1, 0),
+    (8, 2, 4),
+]
+
+log_levels = (
+    "debug",
+    "info",
+    "warning",
+    "error",
+    "critical",
+)
+
+# define sources of version strings
+RELEASE_TAG = os.getenv("RELEASE_TAG")
+PROJECT_VERSION = report.__version__
+
+
+def test_stdout_version(capsys):
+    """Verify that version string sent to stdout agrees with the module version."""
+    with pytest.raises(SystemExit):
+        with patch.object(sys, "argv", ["bogus", "--version"]):
+            report.main()
+    captured = capsys.readouterr()
+    assert (
+        captured.out == f"{PROJECT_VERSION}\n"
+    ), "standard output by '--version' should agree with module.__version__"
+
+
+# def test_running_as_module(capsys):
+#     """Verify that the __main__.py file loads correctly."""
+#     with pytest.raises(SystemExit):
+#         with patch.object(sys, "argv", ["bogus", "--version"]):
+#             # F401 is a "Module imported but unused" warning. This import
+#             # emulates how this project would be run as a module. The only thing
+#             # being done by __main__ is importing the main entrypoint of the
+#             # package and running it, so there is nothing to use from this
+#             # import. As a result, we can safely ignore this warning.
+#             # Third-Party Libraries
+#             import report.__main__  # noqa: F401
+#     captured = capsys.readouterr()
+#     assert (
+#         captured.out == f"{PROJECT_VERSION}\n"
+#     ), "standard output by '--version' should agree with module.__version__"
```

Comment:
This test should be re-enabled and made to work with this project.

Reply:
Resolved in 1162f92

Reply:
This looks good now, thanks!